### ISSUE TITLE
Published npm version should contain src folder

### ISSUE THREAD

**Issue by EugeneSnikhovskiy** [atulmy/gql-query-builder#64]
Hello. First of all thanks for great tool!

I updated my app to webpack 5 and it got smarter :)

For now I receive warnings that webpack can't find source maps for gql-query-builder
```bash
Failed to parse source map from '/home/eugene/projects/front-tm2/node_modules/gql-query-builder/src/NestedField.ts' file: Error: ENOENT: no such file or directory, open '/home/eugene/projects/front-tm2/node_modules/gql-query-builder/src/NestedField.ts'

Failed to parse source map from '/home/eugene/projects/front-tm2/node_modules/gql-query-builder/src/OperationType.ts' file: Error: ENOENT: no such file or directory, open '/home/eugene/projects/front-tm2/node_modules/gql-query-builder/src/OperationType.ts'

Failed to parse source map from '/home/eugene/projects/front-tm2/node_modules/gql-query-builder/src/Utils.ts' file: Error: ENOENT: no such file or directory, open '/home/eugene/projects/front-tm2/node_modules/gql-query-builder/src/Utils.ts'

Failed to parse source map from '/home/eugene/projects/front-tm2/node_modules/gql-query-builder/src/adapters/DefaultAppSyncMutationAdapter.ts' file: Error: ENOENT: no such file or directory, open '/home/eugene/projects/front-tm2/node_modules/gql-query-builder/src/adapters/DefaultAppSyncMutationAdapter.ts'

Failed to parse source map from '/home/eugene/projects/front-tm2/node_modules/gql-query-builder/src/adapters/DefaultAppSyncQueryAdapter.ts' file: Error: ENOENT: no such file or directory, open '/home/eugene/projects/front-tm2/node_modules/gql-query-builder/src/adapters/DefaultAppSyncQueryAdapter.ts'

Failed to parse source map from '/home/eugene/projects/front-tm2/node_modules/gql-query-builder/src/adapters/DefaultMutationAdapter.ts' file: Error: ENOENT: no such file or directory, open '/home/eugene/projects/front-tm2/node_modules/gql-query-builder/src/adapters/DefaultMutationAdapter.ts'

Failed to parse source map from '/home/eugene/projects/front-tm2/node_modules/gql-query-builder/src/adapters/DefaultQueryAdapter.ts' file: Error: ENOENT: no such file or directory, open '/home/eugene/projects/front-tm2/node_modules/gql-query-builder/src/adapters/DefaultQueryAdapter.ts'

Failed to parse source map from '/home/eugene/projects/front-tm2/node_modules/gql-query-builder/src/adapters/DefaultSubscriptionAdapter.ts' file: Error: ENOENT: no such file or directory, open '/home/eugene/projects/front-tm2/node_modules/gql-query-builder/src/adapters/DefaultSubscriptionAdapter.ts'

Failed to parse source map from '/home/eugene/projects/front-tm2/node_modules/gql-query-builder/src/adapters/index.ts' file: Error: ENOENT: no such file or directory, open '/home/eugene/projects/front-tm2/node_modules/gql-query-builder/src/adapters/index.ts'

Failed to parse source map from '/home/eugene/projects/front-tm2/node_modules/gql-query-builder/src/index.ts' file: Error: ENOENT: no such file or directory, open '/home/eugene/projects/front-tm2/node_modules/gql-query-builder/src/index.ts'
```
And, for example `node_modules/gql_query_builder/build/Fields.js.map` has this code
```json
{"version":3,"file":"Fields.js","sourceRoot":"","sources":["../src/Fields.ts"],"names":[],"mappings":""}
```
`"sources":["../src/Fields.ts"]` points to a non-existent file

**Comment by EugeneSnikhovskiy** [comment]
I see `help wanted` label.
I don't know how you publish new versions, but I can assume that with `npm publish` npm command,
and probably you have `.npmignore` file in project root folder.
If yes - you need to remove or comment `src` line

**Comment by atulmy** [comment]
Fixed by #69
